account_asset_management_xls/wizard/wiz_account_asset_report.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2009-2018 Noviat
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import unicodedata
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class WizAccountAssetReport(models.TransientModel):
+    _name = 'wiz.account.asset.report'
+    _description = 'Financial Assets report'
+
+    # TODO: add support for all date range types
+    date_range_id = fields.Many2one(
+        comodel_name='date.range',
+        string='Fiscal Year',
+        domain=[('type_id.fiscal_year', '=', True)],
+        required=True)
+    parent_asset_id = fields.Many2one(
+        comodel_name='account.asset',
+        string='Asset Filter', domain=[('type', '=', 'view')])
+
+    @api.multi
```

Comment:
What is the point of using `api.multi` if you are going to use `self.ensure_one()`? I assume that no one will call this function using the old api. https://github.com/OCA/OCB/blob/12.0/odoo/api.py#L677